src/components/Button/Button.module.scss

```diff
@@ -64,9 +64,6 @@ button,
     padding-right: 48px;
     padding-left: 48px;
   }
-  @include for-phone-only {
```

Comment:
Ah ok, I did https://github.com/getethos/ethos-design-system/commit/20ee3c593219c8d70bea2cbba5f56cc0fbafb9c8#diff-f8f2965423d01560e3bd62a4baabb6d1R66 because when I introduced CSS Modules I found this in CMS and removed. Are we certain it's not needed? I'm all for this if we're sure cc / @zach-ethos @kenning